src/varname.jl

```diff
@@ -1,6 +1,6 @@
 using Accessors
 using Accessors: ComposedOptic, PropertyLens, IndexLens, DynamicIndexLens
-using MacroTools
+using Accessors.MacroTools: @capture
```

Comment:
Accessors may remove the dependency on MacroTools at any time in a non-breaking release. Additionally, it may switch to a new breaking release of MacroTools and we can't prevent this from breaking `@capture`. So IMO we should either add a compat entry for MacroTools (probably easiest short term solution) or rewrite `@capture` and remove MacroTools completely (probably best long term approach).